pkg/python/testdata/requirements_flask.txt

```diff
@@ -0,0 +1,8 @@
+click==8.0.0
+Flask==2.0.0
+itsdangerous==2.0.0
```

Comment:
https://pip.pypa.io/en/latest/cli/pip_install/#requirements-file-format
The following patterns should also be supported.
```
itsdangerous==2.0.0 # a comment 
# itsdangerous==2.0.0
```

Reply:
Added comment stripping and test coverage for this. Let me know if further test cases are required.